gk/main.c

```diff
@@ -1292,6 +1496,8 @@ cleanup_gk(struct gk_config *gk_conf)
 				ip_flow_entry_table);
 		}
 
+		rte_timer_stop(&gk_conf->instances[i].scan_timer);
```

Comment:
I understand that the timers were all initialized in their own loop in `gk_stage1()`. I don't think we need to give `rte_timer_stop()` its own loop here, but shouldn't we at least put the call to `rte_timer_stop()` after the call to `destroy_mailbox()` in this loop?